echo/udp-echo/client/assembly/common/build.sh

```
@@ -11,28 +11,28 @@
+version=$(cat app/version.go | grep Version | awk -F '=' '{print $2}' | awk -F '"' '{print $2}')
 if [[ ${GOOS} == "windows" ]]; then
     TARGET_SBIN_NAME=${TARGET_SBIN_NAME}.exe
 fi
 TARGET_NAME=${TARGET_FOLDER}/${TARGET_SBIN_NAME}
-if [[ $PROFILE == "dev" ||  $PROFILE == "test" ]]; then
+if [[ $PROFILE == "dev" || $PROFILE == "test" ]]; then
     # GFLAGS=-gcflags "-N -l" -race -x -v # -x会把go build的详细过程输出
     # GFLAGS=-gcflags "-N -l" -race -v
     # GFLAGS="-gcflags \"-N -l\" -v"
-    cd ${BUILD_PACKAGE} && GOOS=$GOOS GOARCH=$GOARCH go build -gcflags "-N -l" -x -v -i -o ${TARGET_NAME} && cd -
+    cd ${BUILD_PACKAGE} && GOOS=$GOOS GOARCH=$GOARCH go build -gcflags "-N -l" -x -v -o ${TARGET_NAME} && cd -
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)